src/gui/sdlmain.cpp

```diff
@@ -2876,6 +2880,10 @@ void Config_Add_SDL() {
 	Pint = Pmulti->GetSection()->Add_int("ysens",Property::Changeable::Always,100);
 	Pint->SetMinMax(-1000,1000);
 
+	Pbool = sdl_sec->Add_bool("rawmouseinput",
+	                          Property::Changeable::OnlyAtStart, false);
+	Pbool->Set_help("Mouse is routed through OS API by default. Enable this to poll mouse directly which bypasses OS mouse settings.");
```

Comment:
Regarding this statement _"Enable this to poll mouse directly which bypasses OS"_, I don't think SDL is running at a ring lower than the OS and has direct hardware access - does it? 

As far as I know, SDL _only_ knows how to interface with the OS, and goes no further. Can we improve/clarify the distinction (without bogging users down in SDL or developer jargon?)

Reply:
Honestly, I don't actually know the semantics of how raw input works. So I'm not sure how else to phrase this.

Reply:
Found a Microsoft article: https://docs.microsoft.com/en-us/windows/win32/inputdev/about-raw-input

Reply:
Can you investigate the WARP variable from SDL's perspective, as that's a what's we're calling (we're not by-passing SDL).  SDL's description should also be relatively OS-agnotic, because presumabling the WARP* variable is valid on non-Windows OSes as well.

Reply:
It most likely does what the article described which is register device for raw input under Windows. So that wouldn't help much. We should focus on explaining what this variable is for and go from there. The purpose of raw input on the user side is make mouse behavior under DosBox independent from OS mouse settings so that's what should be conveyed in the description.

Reply:
@kcgen I thought about this and I don't know how else to phrase it. The current description is rough but it conveys the gist of the feature to the normie, IMHO.